MvvmCross/MvvmCross.csproj

```diff
@@ -77,5 +84,6 @@ This package contains the 'Core' libraries for MvvmCross</Description>
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" />
+    <PackageReference Include="PInvoke.User32" />
```

Comment:
Please move this into the appropriate WinUI related ItemGroup, otherwise you are adding this dependency for all platforms, which is not needed.

Reply:
Done.